classes/classes/BodyParts/Claws.as

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by aimozg on 27.04.2017.
+ */
+package classes.BodyParts {
+import classes.Creature;
+
+public class Claws extends BodyPart {
+	public static const TYPE_NORMAL:int     = 0;
+	public static const TYPE_LIZARD:int     = 1;
+	public static const TYPE_DRAGON:int     = 2;
+	public static const TYPE_SALAMANDER:int = 3;
+	public static const TYPE_CAT:int        = 4; // NYI! Placeholder for now!! (See http://tiny.cc/coc-revamp-claws)
+	public static const TYPE_DOG:int        = 5; // NYI! Placeholder for now!! (See http://tiny.cc/coc-revamp-claws)
+	public static const TYPE_RAPTOR:int     = 6; // NYI! Placeholder for now!! (See http://tiny.cc/coc-revamp-claws) Giev teh Rapturs :-)
+	public static const TYPE_MANTIS:int     = 7; // NYI! Placeholder for Xianxia mod (See http://tiny.cc/coc-xianxia-mod)
+
+	public var tone:String                  = "";
+
+	public function Claws(creature:Creature) {
+		super(creature);
```

Comment:
For the sake of unification.
Leaving it in superclass and making it default null is no options (I'll need to manually assign it for each body part that will need it, and will have it in completion in the parts that don't).
So it will be either in each body part (automatically) once and for all, or should added manually when needed to each body part, but in the same manner.